src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -294,10 +294,10 @@ public float getAverageMessageLength() {
 
   public Set<String> getUsers() {
     Set<String> users = new HashSet<>();
-    Query query = new Query("Message");
+    Query query = new Query("User");
```

Comment:
I think this function is used in other cases like get all the users who have posted messages before. If change query to `user` then it will return all registered users.

Im not sure this function is being called in other functions, but it will be safer if you create a seperate function.

Reply:
I've checked I see that this function didn't work for community page so I changed it. But I think you're right, I've created a separate function.

Reply:
Great! Since we are not sure if this function will be used in the future or currently locally developped branches.